app/lib/features/events/sheets/edit_event_sheet.dart

```diff
@@ -165,41 +164,6 @@ class _EditEventSheetConsumerState extends ConsumerState<EditEventSheet> {
                   ),
                 ),
                 const SizedBox(width: 15),
-                Expanded(
-                  child: Column(
-                    crossAxisAlignment: CrossAxisAlignment.start,
-                    children: <Widget>[
-                      const Padding(
-                        padding: EdgeInsets.only(bottom: 5),
-                        child: Text('End Time'),
-                      ),
-                      InkWell(
-                        focusColor: Colors.transparent,
-                        hoverColor: Colors.transparent,
-                        onTap: _selectDateTime,
-                        child: TextFormField(
-                          enabled: false,
-                          controller: _endTimeController,
-                          keyboardType: TextInputType.datetime,
-                          style: Theme.of(context).textTheme.labelLarge,
-                          decoration: InputDecoration(
-                            fillColor:
-                                Theme.of(context).colorScheme.primaryContainer,
-                            filled: true,
-                            hintText: 'Select Time',
-                            border: OutlineInputBorder(
-                              borderSide: BorderSide.none,
-                              borderRadius: BorderRadius.circular(12),
-                            ),
-                            focusedBorder: OutlineInputBorder(
-                              borderRadius: BorderRadius.circular(12),
-                            ),
-                          ),
-                        ),
-                      ),
-                    ],
-                  ),
-                ),
               ],
             ),
             const SizedBox(height: 15),
```

Comment:
so we have a SizedBox above a SizedBox here now? That seems odd...

Reply:
Ahh might have been missed along with other removal of code-blocks....